emscripten.py

```diff
@@ -97,7 +97,7 @@ def emscript_fastcomp(infile, outfile, memfile, libraries, compiler_engine,
 
     with ToolchainProfiler.profile_block('function_tables_and_exports'):
       (post, function_table_data, bundled_args) = (
-          function_tables_and_exports(funcs, metadata, mem_init, glue, forwarded_data, outfile, DEBUG))
+          function_tables_and_exports(funcs, metadata, mem_init, glue, forwarded_data, outfile, DEBUG)) 
```

Comment:
This line looks like it can be reverted?

Reply:
Yes, I will revert the extra space here.